change-address/REPWRITERSPECS/BANNO.CHANGE.ADDR.V1.POW

```diff
@@ -276,6 +306,475 @@ PRINT TITLE="BANNO.CHANGE.ADDR.V1.POW"
+                               VALIDNAMETYPES(SHARE NAME:TYPE)=TRUE AND
+                               SHARE NAME:STREET=OLDSTREET AND
+                               SHARE NAME:EXTRAADDRESS=OLDEXTRA)
+      DO
+       IF SHARENAMECOUNT<NAMECOUNTMAX THEN
+        DO
+         TOTALNAMECOUNT=TOTALNAMECOUNT+1
+         SHARENAMECOUNT=SHARENAMECOUNT+1
+         SHARELVLLOCS(SHARENAMECOUNT)=SHARE NAME:LOCATOR
+         SHAREID(SHARENAMECOUNT)=SHARE:ID
+        END
+      END
+    END
+  END
+ IF MATCHIRSNAMES=TRUE THEN
+  DO
+   FOR EACH IRS
+    DO
+     FOR EACH IRS NAME WITH (IRS NAME:EXPIRATIONDATE=DATENULL AND
+                             IRS NAME:DEATHDATE=DATENULL AND
+                             VALIDNAMETYPES(IRS NAME:TYPE)=TRUE AND
+                             IRS NAME:STREET=OLDSTREET AND
+                             IRS NAME:EXTRAADDRESS=OLDEXTRA)
+      DO
+       IF IRSNAMECOUNT<NAMECOUNTMAX THEN
+        DO
+         TOTALNAMECOUNT=TOTALNAMECOUNT+1
+         IRSNAMECOUNT=IRSNAMECOUNT+1
+         IRSLVLLOCS(IRSNAMECOUNT)=IRS NAME:LOCATOR
+         IRSLOC(IRSNAMECOUNT)=IRS:LOCATOR
+        END
+      END
+    END
+  END
+ IF MATCHLOANNAMES=TRUE OR 
+    MATCHPLEDGENAMES=TRUE THEN
+  DO
+   FOR EACH LOAN WITH (LOAN:CLOSEDATE=DATENULL)
+    DO
+     IF MATCHLOANNAMES=TRUE THEN
+      DO
+       FOR EACH LOAN NAME WITH (LOAN NAME:EXPIRATIONDATE=DATENULL AND
+                                LOAN NAME:DEATHDATE=DATENULL AND
+                                VALIDNAMETYPES(LOAN NAME:TYPE)=TRUE AND
+                                LOAN NAME:STREET=OLDSTREET AND
+                                LOAN NAME:EXTRAADDRESS=OLDEXTRA)
+        DO
+         IF LOANNAMECOUNT<NAMECOUNTMAX THEN
+          DO
+           TOTALNAMECOUNT=TOTALNAMECOUNT+1
+           LOANNAMECOUNT=LOANNAMECOUNT+1
+           LOANLVLLOCS(LOANNAMECOUNT)=LOAN NAME:LOCATOR
+           LOANID(LOANNAMECOUNT)=LOAN:ID
+          END
+        END
+      END
+     IF MATCHPLEDGENAMES=TRUE THEN
+      DO
+       FOR EACH LOAN PLEDGE
+        DO
+         FOR EACH LOAN PLEDGE NAME WITH (LOAN PLEDGE NAME:EXPIRATIONDATE=DATENULL AND
+                                         LOAN PLEDGE NAME:DEATHDATE=DATENULL AND
+                                         VALIDNAMETYPES(LOAN PLEDGE NAME:TYPE)=TRUE AND
+                                         LOAN PLEDGE NAME:STREET=OLDSTREET AND
+                                         LOAN PLEDGE NAME:EXTRAADDRESS=OLDEXTRA)
+          DO
+           IF PLEDGENAMECOUNT<NAMECOUNTMAX THEN
+            DO
+             TOTALNAMECOUNT=TOTALNAMECOUNT+1
+             PLEDGENAMECOUNT=PLEDGENAMECOUNT+1
+             PLEDGELVLLOCS(PLEDGENAMECOUNT)=LOAN PLEDGE NAME:LOCATOR
+             PLEDGELOC(PLEDGENAMECOUNT)=LOAN PLEDGE:LOCATOR
+             PLEDGELOANID(PLEDGENAMECOUNT)=LOAN:ID
+            END
+          END
+        END
+      END
+    END
+  END
+ IF MATCHEFTNAMES=TRUE THEN
+  DO
+   FOR EACH EFT WITH (EFT:EXPIRATIONDATE<SYSTEMDATE OR
```

Comment:
Instead of "EFT:EXPIRATIONDATE<SYSTEMDATE" I think you mean "EFT:EXPIRATIONDATE>SYSTEMDATE" ?

Reply:
Good catch! Thanks. Done.